packages/wallet-sdk/src/CoinbaseWalletSDK.test.ts

```diff
@@ -2,28 +2,32 @@ import { CoinbaseWalletProvider } from './CoinbaseWalletProvider';
 import { CoinbaseWalletSDK } from './CoinbaseWalletSDK';
 import { ProviderInterface } from ':core/provider/interface';
 import { getFavicon } from ':core/type/util';
-import { getCoinbaseInjectedProvider } from ':util/provider';
+import { CBWindow } from ':util/provider';
+
+const window = globalThis as CBWindow;
 
 jest.mock(':core/type/util');
 jest.mock(':util/provider');
 jest.mock('./CoinbaseWalletProvider');
+class MockCipherProviderClass {
+  public isCoinbaseBrowser = true;
+}
+
+const mockCipherProvider = new MockCipherProviderClass() as unknown as ProviderInterface;
```

Comment:
i believe you can create a mocked instance object without defining a class

Reply:
Ah yeah thanks! Had copied this from the provider.test.ts - cleaned up here [70e49b5](https://github.com/coinbase/coinbase-wallet-sdk/pull/1332/commits/70e49b5db02ae2716c095638912fe300ce96f162)